src/deserializer.rs

```diff
@@ -21,11 +21,13 @@ use self::DeserializeErrorKind as DEK;
 pub fn deserialize_string_record<'de, D: Deserialize<'de>>(
     record: &'de StringRecord,
     headers: Option<&'de StringRecord>,
+    missing_field_check: Option<&'de dyn Fn(&str) -> bool>,
```

Comment:
This looks like a breaking change.  New argument to a public function.

Reply:
hmm let me have a think